tests/test_deployment.py

```diff
@@ -6,45 +11,66 @@
 
 
 def test_minimal_install(clean_docker_main):
+    # Create tmpdir in container to make sure local universum sources are not used instead of installed
+    tmpdir = os.path.join(clean_docker_main.working_dir, randomize_name('tmpdir'))
+    clean_docker_main.environment.assert_successful_execution(f"mkdir '{tmpdir}'")
+
     # Run without parameters
-    log = clean_docker_main.environment.assert_unsuccessful_execution("universum")
-    assert "command not found" not in log
+    log = clean_docker_main.environment.assert_unsuccessful_execution("python3.7 -m universum")
```

Comment:
This block and the cleanup block should be changed to the fixture.
Probably they can be merged into clean_docker fixture.